src/constants/constants.ts

```diff
@@ -38,6 +27,18 @@ export const ACTIVATION_EXIT_DELAY = 2 ** 2; // epochs || 25.6 minutes
 export const EPOCHS_PER_ETH1_VOTING_PERIOD = 2 ** 4; // epochs || ~1.7 hours
 export const MIN_VALIDATOR_WITHDRAWAL_DELAY = 2 ** 8; // epochs || ~27 hours
 
+// Initial values
+export const GENESIS_FORK_VERSION	= new BN(0);
+export const GENESIS_SLOT = 0;
+export const SLOT_OFFSET = 2 ** 32; // 2 ** 32
```

Comment:
2 ** 63?

Reply:
Ah yeah, I was already looking ahead, lol

Reply:
actually, there's a problem using 2 ** 63 right now. Our ssz currently assumes a `number` offset.
We should change it in any case to allow for a BN offset, but for now this variable has to be a number.

Reply:
Ahhhh...